git_machete/github.py

```diff
@@ -16,7 +16,6 @@
 
 
 GITHUB_TOKEN_ENV_VAR = 'GITHUB_TOKEN'
-
```

Comment:
Whoops does it pass pep8?

Reply:
yes, what is wrong here?

Reply:
Nvm, I thought newline is strictly required in this place